### PR TITLE
add working ping test (js) or multidim-interop

### PR DESCRIPTION
**Pull request by GlenDC**
You can test it yourself:

```
# terminal A. Get redis running
docker run --rm -it -p 6379:6379 redis/redis-stack

# terminal B (in js/v0.41)
REDIS_ADDR=localhost:6379 ip="0.0.0.0" transport=ws security=noise muxer=mplex is_dialer="true" node index.js

# terminal C (in js/v0.41)
REDIS_ADDR=localhost:6379 ip="0.0.0.0" transport=ws security=noise muxer=mplex is_dialer="false" node index.js
```

Haven't hooked it up to Go/Rust yet. Let me know if this PR suffices for you or if you need me to undertake some more steps.

In function of https://github.com/libp2p/test-plans/pull/97.

### CODE REVIEW

**Comment by GlenDC** [comment]
@MarcoPolo this PR is ready for review what is me concerned.

File: multidim-interop/js/v0.41/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+WORKDIR /app
+
+COPY package*.json .
+
+RUN npm ci
+
+COPY *.js /app/
+
+EXPOSE 6060
```
Comment (MarcoPolo) [inline multidim-interop/js/v0.41/Dockerfile:13]: Un-needed (I know this is in the other dockerfiles)

File: multidim-interop/js/v0.41/index.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    switch (SECURE_CHANNEL) {
+        case 'noise':
+            options.connectionEncryption = [noise()]
+            break
+        default:
+            throw new Error(`Unknown secure channel: ${TRANSPORT}`)
+    }
+
+    switch (MUXER) {
+        case 'mplex':
```
Comment (MarcoPolo) [inline multidim-interop/js/v0.41/index.js:59]: What about yamux?

**Review by MarcoPolo — APPROVED** [review]
Some small comments. Looks great! I'll merge this soon.

In the mean time, please start the browser version. I'm imagining this will be almost identical to this node version, except once you have the listener addr, you start the playwright instance and connect.

**Comment by marten-seemann** [comment]
Super happy to see this happen! The code looks really clean and straightforward, which is a big plus for the wo-testground architecture.

**Comment by MarcoPolo** [comment]
Merging this now, then merging #97 when CI passes :)

**Pull request merged**